DOCKER/Dockerfile

```diff
@@ -15,9 +16,34 @@ RUN apt-get update && \
   rm -rf /var/lib/apt/lists/*
 
 # set the repo and install tendermint
-ENV repo /go/src/github.com/eris-ltd/eris-db
-ADD . $repo
-WORKDIR $repo
-RUN cd ./cmd/erisdb && go install
-USER eris
-ENTRYPOINT ["erisdb"]
+WORKDIR $GOPATH/src/
+ENV REPO github.com/tendermint/tendermint
+ENV BRANCH permissions
+RUN mkdir --parents $REPO
+WORKDIR $GOPATH/src/$REPO
+RUN git clone https://$REPO . && \
+  git checkout $BRANCH && \
+  make && \
+  mv ./build/* /usr/local/bin/
+
+# set the repo and install erisdb
+ENV REPO /go/src/github.com/eris-ltd/eris-db
+COPY DOCKER/start.sh /usr/local/bin/erisdb-wrapper
+COPY . $REPO
+WORKDIR $REPO
+RUN cd ./cmd/erisdb && \
+  go build && \
+  mv erisdb /usr/local/bin/ && \
+  cd ../erisdbss && \
+  go build && \
+  mv erisdbss /usr/local/bin/
+
+# persist data, set user
+VOLUME /home/$USER/.eris
+WORKDIR /home/$USER/.eris
+USER $USER
+RUN mkdir --parents /home/$USER/.eris/blockchains/tendermint
+ENV TMROOT /home/$USER/.eris/blockchains/tendermint
```

Comment:
sure. but it's just a default. since the whole `.eris` is in a volume we can use data containers there and then just set the TMROOT env var in the chain definition file.... at least that was my thinking....